pyproject.toml

```diff
@@ -153,6 +153,10 @@ datasets-endpoint = [
     "jsonschema",
     "pynamodb",
 ]
+dataset_versions-endpoint = [
```

Comment:
This extra needs to be mentioned where we run `poetry install` with all the other extras (GitHub Actions, reset-dev-env.bash), even if it overlaps with the others right now.